src/test/java/com/wikia/webdriver/common/driverprovider/NewDriverProvider.java

```diff
@@ -225,9 +220,8 @@ private static EventFiringWebDriver getChromeInstance() {
     }
 
     System.setProperty("webdriver.chrome.driver",
-                       new File(ClassLoader.getSystemResource("ChromeDriver" + chromeBinaryPath)
-                                    .getPath())
-                           .getPath());
+        new File(ClassLoader.getSystemResource("ChromeDriver" + chromeBinaryPath).getPath())
+            .getPath());
 
     // TODO change mobile tests to use @UserAgent annotation
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)